apps/opik-backend/src/main/java/com/comet/opik/infrastructure/bi/Metadata.java

```diff
@@ -0,0 +1,6 @@
+package com.comet.opik.infrastructure.bi;
+
+enum Metadata {
+    anonymous_id,
```

Comment:
Please follow the usual Java convention for the enum name and provide a field to resolve the `anonymous_id` literal.